### DISCUSSION TITLE
Active?

### DISCUSSION THREAD

**Discussion by cclcng** [dozius/DtBlkFx#10] [General]
Is there still active development happening on this wonderful project?. it continues to be one of he most exciting pieces of software ever for me.. I wish I knew how to code and/or help here.. It looks like there was an attempt at porting to juce?
Anyways, thank everyone involved (dozius?) for creating dtblkfx... 
if you need someone totest any new builds look no further :) 
AND HONESTLY ,  A LITTLE INSTABILITY / BROKENNESS  WOULD PROBABLY AUGMENT THE PLUGIN IN A VERY 
FUN WAY....
THX AGAIN (:

**Comment by dozius** [comment]
I have no plans to work on this myself. [PaulHaeger](https://github.com/PaulHaeger) was working on porting the core DSP over to JUCE but I'm not in regular communication with them. You can follow progress over at #4.
